submissions/Nemooochka/friends-app/js/cards.js

```diff
@@ -0,0 +1,51 @@
+export class Cards {
+    constructor(arrayCards, node) {
+        this.arrayCards = arrayCards;
+        this.node = node;
+
+        this.renderCards();
+    }
+
+    static generateCard({name, gender, email, location, picture, dob}) {
+        return `<div class="card-content">
+                <div class="card-photo" style="background-image: url(${picture.large})"></div>
```

Comment:
use the `img` tag here instead of setting inline styles for this element

Reply:
I use an inline style because of the need for background-cover: cover as I want the image to be with fixed height without strange resize (as the width of the card isn't fixed).

Reply:
ok. it's no big deal